test/couchdb_mrview_cors_tests.erl

```diff
@@ -86,14 +95,25 @@ create_db(backdoor, DbName) ->
     {ok, Db} = couch_db:create(DbName, [?ADMIN_CTX]),
     couch_db:close(Db);
 create_db(clustered, DbName) ->
-    ok = fabric:create_db(DbName, [?ADMIN_CTX]).
+    {ok, Status, _, _} = test_request:put(db_url(DbName), [?AUTH], ""),
+    true = lists:member(Status, [201, 202]),
+    ok.
 
 delete_db(backdoor, DbName) ->
     couch_server:delete(DbName, [?ADMIN_CTX]);
 delete_db(clustered, DbName) ->
-    ok = fabric:delete_db(DbName, [?ADMIN_CTX]).
+    {ok, 200, _, _} = test_request:delete(db_url(DbName), [?AUTH]),
```

Comment:
In the spirit of YAGNI, I would be inclined to assume 200 for successful DELETE response until otherwise necessary. Also, the docs don't mention 202:
http://docs.couchdb.org/en/latest/api/database/common.html?highlight=delete#delete--db